src/main/java/org/solid/testharness/utils/DataModelBase.java

```diff
@@ -65,7 +68,7 @@ protected DataModelBase(final IRI subject, final ConstructMode mode) {
         }
         model = Repositories.graphQuery(dataRepository,
                 String.format(query, subject, subject),
-                r -> QueryResults.asModel(r)
+                QueryResults::asModel
```

Comment:
OMG, my brain dumped an "unknown syntax" error :-D

Reply:
It is a Java 9+ change. It is a functional style call of a method and the 2 statements are identical. The code style checker prefers the functional style hence the change.